spec/yabeda/activejob_spec.rb

```diff
@@ -47,19 +49,17 @@
         .with(be_between(0.005, 0.05))
     end
 
-    it "measures job latency" do
-      ActiveJob::Base.queue_adapter = :test
+    it "measures job latency", queue_adapter: :test do
       expect { HelloJob.perform_later }.to have_enqueued_job.on_queue("default")
-      sleep(1)
       expect { perform_enqueued_jobs }.to measure_yabeda_histogram(Yabeda.activejob.latency)
         .with_tags(queue: "default", activejob: "HelloJob", executions: "1")
-        .with(be_between(1, 2))
+        .with(kind_of(Float))
     end
 
     context "when enqueued_at is not present" do
-      it "does not measure job latency" do
-        ActiveJob::Base.queue_adapter = :test
-        expect_any_instance_of(HelloJob).to receive(:enqueued_at).and_return(nil).twice # rubocop:disable RSpec/AnyInstance
+      it "does not measure job latency", queue_adapter: :test do
```

Comment:
Could you add a quick test similar to this one when enqueued_at returns a Time object to confirm that stuff still works?

Reply:
We already have test just above this one.

Reply:
ooh amazing i just needed to scroll up! 💚